domain/consensus/processes/consensusstatemanager/resolve_virtual_test.go

```diff
@@ -311,10 +435,10 @@ func printUtxoDiffChildren(t *testing.T, hashes []*externalapi.DomainHash, tc te
 			if err != nil {
 				t.Fatalf("Error while reading utxo diff store: %+v", err)
 			}
-			fmt.Printf("%s\t\t\t%s\n", blocks[*block], blocks[*utxoDiffChild])
+			printfDebug("%s\t\t\t%s\n", blocks[*block], blocks[*utxoDiffChild])
```

Comment:
I wonder if t.Log is sufficient to what you are using printfDebug for.

t.Logf only prints when the test fails, so it seems convenient, and it's very annoying if for the debugging of a test you should go and uncomment stuff, especially if you are not the one who wrote the test.

Reply:
t.Log does not print cleanly. This is "debug" info in the real sense, I see no reason to delete it all together

Reply:
I'm not sure I love it, but can't suggest anything better, and this is not critical, so fine for now. If I have a better idea I might do it myself.